corehq/apps/locations/fixtures.py

```diff
@@ -79,6 +84,17 @@ def __call__(self, restore_user, version, last_sync=None, app=None):
         if not should_sync_locations(last_sync, all_locations, restore_user):
             return []
 
+        xml_nodes = []
+        for serializer in self.serializers:
+            xml_nodes.extend(serializer.get_xml_nodes(restore_user, all_locations))
```

Comment:
yeah, I imagine if any projects with a large list of locations want this, they'll probably want only one so the restore payload is smaller, but @sheelio can probably comment more conclusively.